Feature/create

tsohleDev opened the pull request:
**Create React App**

- Create and assemble Components
- Integrate the API
- Create a routable environment
- Write unit tests

tsohleDev committed:
Add Cdn packages for google icons

tsohleDev committed:
Write test

tsohleDev committed:
Download picture

tsohleDev committed:
Add components

tsohleDev committed:
Add Redux-react

tsohleDev committed:
Update README to include link to live demo

omarsalem7 commented on README.md:
```diff
@@ -1,65 +1,219 @@
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Credits to everyone who inspired the codebase.
+
+I would like to thank [Microverse](https://microverse.org) for the great content
+
```
- [x] Kindly give appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

omarsalem7 commented on src/__test__/home.test.js:
```diff
@@ -0,0 +1,19 @@
+import store from '../redux/configureStore';
+import Home from '../components/home/home';
+
+describe('Home Test', () => {
+  test('display pairs', () => {
+    const home = render(
+      <Provider store={store}>
+        <BrowserRouter>
+          <Home />
+        </BrowserRouter>
+      </Provider>,
+    );
+    expect(home).toMatchSnapshot();
+  });
```
- [x] Kindly add more tests for the rest of the components of your app 
- [x] Also create integration tests for your application using the React Testing Library.
   - You may need to mock the access to the API, so that your tests don't send actual requests.
   - You may need to mock the connection to the Redux Store.

omarsalem7 reviewed (changes requested):
Hi @tsohleDev 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

![you-are-near](https://media.giphy.com/media/j7vckgz15cAYmxatzB/giphy.gif)

## Highlights
- The project deployed properly ✔️
- The project is a single page application (SPA) using `react-router-dom` ✔️

## Required Changes ♻️

- [x] Kindly recorded a video presenting the features of the project. The content of the video should include:
    - a description of the project.
    - a demo of the project features.
    - you should also highlight some interesting piece of code or something you built that you are very proud of.




- [x] _Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

tsohleDev committed:
More tests and update README

Write tests for Title, Stock, and the integration test that mock the API
Include the Sakwa on the README and the video of me going through the project

tsohleDev committed:
Fix format in README file

victorokoroji reviewed (approved):
Hi @tsohleDev, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- All test passed ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 
<img src="https://media3.giphy.com/media/xT0xezQGU5xCDJuCPe/giphy.gif"/>

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
